src/applications/caregivers/tests/unit/actions/fetchMapBoxGeocoding.unit.spec.jsx

```diff
@@ -41,7 +41,7 @@ describe('CG fetchMapBoxGeocoding action', () => {
 
     it('should return a `not found` response when coordinates do not match any location', async () => {
       clientStub.returns({
-        send: sinon.stub().resolves({ body: { features: [] } }),
+        send: sinon.stub().resolves({ body: { features: undefined } }),
```

Comment:
This update allows for a conditional to run that sets `features` to an empty array if the value is undefined.